cluster/cluster.go

```diff
@@ -61,6 +61,13 @@ type RemoteBindingListener interface {
 	OnUserBind(uid, fid string)
 }
 
+// InfoRetriever gets cluster info
```

Comment:
Is it cluster or region info?

Reply:
The intension is to be extendable to new values, not only region